src/librawspeed/metadata/ColorFilterArray.h

```diff
@@ -27,16 +27,15 @@
 namespace RawSpeed {
 
 enum CFAColor {
-  CFA_COLOR_MIN = 0,
+  // see also DngDecoder
   CFA_RED = 0,
   CFA_GREEN = 1,
   CFA_BLUE = 2,
-  CFA_GREEN2 = 3,
+  CFA_INVALID = 3, // keep the unit tests happy
```

Comment:
it's not invalid though
The problem is that first 4 colors of CFAColor are used for 2 different purposes
* to enumerate up to 4 different colors in bayer CFA
* to actually specify R/G/B colors from config